chapters/en/chapter5/fine-tuning_parakeet.mdx

```diff
@@ -0,0 +1,361 @@
+    references.append(sample["text"].lower())
+
+error_rate = wer(references, predictions)
+print(f"Word Error Rate: {error_rate:.2%}")
+```
+
+<Tip>
+For Spanish and other languages, it's good practice to normalize the text before computing WER. 
+This includes lowercasing, removing punctuation, and handling special characters consistently.
+</Tip>
+
+## Inference
+
+To use your fine-tuned model for inference:
+
+```python
+import torch
+from transformers import AutoModelForCTC, AutoProcessor
+
+# Load your fine-tuned model
+model_id = "YOUR_USERNAME/parakeet-spanish-lora"
+model = AutoModelForCTC.from_pretrained(model_id, device_map="cuda")
+processor = AutoProcessor.from_pretrained(model_id)
+
+# Load an audio file
+import librosa
+
+audio_array, sr = librosa.load("path/to/spanish_audio.wav", sr=16000)
+
+# Process and transcribe
+inputs = processor(audio=audio_array, sampling_rate=16000, return_tensors="pt")
+inputs = inputs.to("cuda")
+
+with torch.no_grad():
+    logits = model(**inputs).logits
+
+predicted_ids = torch.argmax(logits, dim=-1)
+transcription = processor.batch_decode(predicted_ids)[0]
+
+print(f"Transcription: {transcription}")
+```
+
+You can also test directly on samples from the dataset:
+
+```python
+# Test on a sample from MLS
+sample = next(iter(mls_test))
+print(f"Reference: {sample['text']}")
+
+transcription = transcribe(sample["audio"]["array"])
+print(f"Prediction: {transcription}")
+```
+
+## Using Other Languages
+
+The approach shown in this tutorial can be applied to any language available in Multilingual LibriSpeech. Here are examples for other languages:
+
+```python
+from datasets import load_dataset, Audio
+
+# French
+french_data = load_dataset("facebook/multilingual_librispeech", "french", split="train")
+
+# German
+german_data = load_dataset("facebook/multilingual_librispeech", "german", split="train")
+
+# Italian
+italian_data = load_dataset("facebook/multilingual_librispeech", "italian", split="train")
+
+# Polish
+polish_data = load_dataset("facebook/multilingual_librispeech", "polish", split="train")
+
+
+# For any language, prepare the data the same way:
+def prepare_dataset(dataset):
+    dataset = dataset.select_columns(["audio", "text"])
+    dataset = dataset.cast_column("audio", Audio(sampling_rate=16000))
+    return dataset
+```
+
+MLS supports 8 languages: **German, Dutch, French, Spanish, Italian, Portuguese, Polish**, and **English**.
+
+For languages not in MLS or with non-Latin scripts (e.g., Arabic, Chinese, Japanese), you may need to consider:
+1. **Other datasets**: Use Common Voice or VoxPopuli for broader language coverage.
+2. **Vocabulary extension**: The Parakeet tokenizer is character-based but trained on English. You may need to extend it.
+3. **Phoneme-based approaches**: Convert text to phonemes using tools like `g2p` before training.
+4. **Alternative models**: Consider models pre-trained on multilingual data like Whisper for non-Latin scripts.
+
```

Comment:
Can then finish off by mentioning fine-tuning on other datasets (which is why I suggested the broader title)

"Finally, check out the [Hugging Face Hub](https://huggingface.co/datasets?modality=modality:audio) for more datasets to fine-tune on!"